Definitions added for Cordova plugin cordova-plugin-app-version

Ritzlgrmft opened the pull request:
I added definitions for the Cordova plugin cordova-plugin-app-version (https://github.com/whiteoctober/cordova-plugin-app-version)

vvakame commented:
*cordova-plugin-app-version/appversion.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/cordova-plugin-app-version
  * http://bower.io/search/?q=cordova-plugin-app-version
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (cordova-plugin-app-version/appversion-tests.ts or others)
* [ ] pass the Travis-CI test?

Ritzlgrmft commented:
I fixed the errors now. I renamed the file according to the naming convention, and the Travis check was successful, too.
But now I don't know how to proceed: Do I have to create a new pull request, or are the commits on my fork already sufficient?

Best regards
Markus (aka Ritzlgrmft)
